tests/functional/param_expand/01-basic.t

```diff
@@ -381,9 +381,9 @@ cmp_ok '19.cylc' <<'__FLOW_CONFIG__'
     [[templates]]
         lang = %(lang)s
 [scheduling]
-    cycling mode = integer
     initial cycle point = 1
     final cycle point = 1
+    cycling mode = integer
```

Comment:
What's the benefit?

Reply:
Are you saying that the order these configs are defined in makes a functional difference?